Justfile

```diff
@@ -53,11 +53,18 @@ build-cdb:
 
 # Build only flox
 @build-cli: build-pkgdb
-    pushd cli; cargo build -q
+    pushd cli; cargo build -q --workspace
+
+# Build just the data generator
+@build-data-gen:
+    pushd cli; cargo build -q -p mk_data; popd
 
 # Build the binaries
 build: build-cli
 
+# Regenerate test data
```

Comment:
suggestion nonblocking: regenerate makes me think this would pass `-f`. Either pass `-f`, or don't say regenerate